libsource/src/main/res/values-ja/strings.xml

```diff
@@ -2,4 +2,15 @@
 <resources>
```

Comment:
Those should come from [i18n api endpoint](https://i18n.virtusize.jp/bundle-payloads/aoyama/ja) and not be static in the app.

Each languages:

- https://i18n.virtusize.jp/bundle-payloads/aoyama/ja
- https://i18n.virtusize.jp/bundle-payloads/aoyama/ko
- https://i18n.virtusize.jp/bundle-payloads/aoyama/en

Reply:
OK, I just created the ticket for it. will work on it and create another PR